.gitpod.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM gitpod/workspace-mysql
+FROM gitpod/workspace-mysql:latest
```

Comment:
The '`:latest` tag is sematically inconsistant and not required. I am going to leave this one out since you can tag an image without anything after the colon and your image will be automatically given latest as its version tag. This is a common source of confusion for newcomers to Docker. Tagging with `:latest` will not give you the most recent build if that build was tagged with something other than latest.
See [The Problems With latest](https://www.cloudsavvyit.com/10691/understanding-dockers-latest-tag/).